packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -245,6 +247,24 @@ class NavigationBar extends StatelessWidget {
   /// the top.
   final EdgeInsetsGeometry? labelPadding;
 
+  /// Specifies whether the underlying [SafeArea] should maintain the bottom
+  /// [MediaQueryData.viewPadding] instead of the bottom [MediaQueryData.padding].
+  ///
+  /// When true, this will prevent the [NavigationBar] from shifting when opening a
+  /// software keyboard due to the change in the padding value, especially when the
+  /// app uses [SystemUiMode.edgeToEdge], which renders the navigation elements over
+  /// the application instead of outside it.
```

Comment:
This is a bit confusing. Maybe "which renders the system bars over the application instead of outside it."?

Reply:
I copied this wording from official `SystemUiMode.edgeToEdge` docs.  It might be too confusing  for `NavigationBar` docs.